reactive-banana/src/Reactive/Banana/Types.hs

```diff
@@ -94,6 +94,13 @@ instance Applicative Behavior where
 instance Functor Behavior where
     fmap = liftA
 
+instance Semigroup a => Semigroup (Behavior a) where
+  (<>) = liftA2 (<>)
+
+instance (Semigroup a, Monoid a) => Monoid (Behavior a) where
```

Comment:
This looks odd to me, I would have expected `Monoid a => Monoid (Behavior a)` and `mappend = liftA2 mappend`. is it some backwards-compat idiom from when semigroups was an independent library?